randovania/games/dread/json_data/Artaria.txt

```diff
@@ -312,7 +308,7 @@ Extra - asset_id: collision_camera_003
   > Near Missile Tank 1
       Morph Ball
 
-> Door to Proto EMMI Introduction; Heals? False
+> Door to Proto EMMI Introduction (Upper); Heals? False
```

Comment:
This change was not done by me, and I think was done so that logic ensures you get it in case it's something useful and you cant get back to it by skipping it

Reply:
I did because it was causing issues to the generator. I think we can do a bit more of requiring the pickup tbh.

Reply:
Resolved